Aptoide/aptoide-client-v8
# Bugfix/asv 378 broken stars on review

**What does this PR do?**

   This PR aims to fix a bug with the rating bar being broken on reviews

**Database changed?**

No

**Where should the reviewer start?**

- [ ] rating_bar_review.xml

**How should this be manually tested?**

Log in, try to review an app, check that the stars are displaying and working correctly

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-378](<https://aptoide.atlassian.net/browse/ASV-378>)

**Code Review Checklist**
